chore: Updated readme instructions for transformIgnorePatterns (#31)
cloudscape-design/jest-preset: README.md

It means that Jest preset could not apply the configuration properly.

> 1. Check that you do not have `transformIgnorePatterns` in your custom configuration. If you do, it requires a manual merge with configuration defined in [jest-preset.js](./jest-preset.js).

Reviewer: I think there's an opportunity to make the first sentence easier to read:
```suggestion
1. Check if you have `transformIgnorePatterns` defined in your custom configuration. If so, it requires a manual merge with configuration defined in [jest-preset.js](./jest-preset.js).
```
